gke-deploy/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:stretch AS build-env
+RUN go get github.com/gobuffalo/packr/v2/packr2
+ADD . /go/src/github.com/GoogleCloudPlatform/cloud-builders/gke-deploy
+RUN go get -d github.com/GoogleCloudPlatform/cloud-builders/gke-deploy
+RUN cd /go/src/github.com/GoogleCloudPlatform/cloud-builders/gke-deploy && packr2
```

Comment:
What does `packr2` do?

Reply:
`packr2` is used to embed static files as go strings. I use it to create in-memory kubernetes objects from files at startup.

Reply:
Hmm, this seems like a fairly large dependency to take on just to embed [these five lines](https://github.com/GoogleCloudPlatform/cloud-builders/blob/c26aeb408bd15b51e9a808958f16988946e8845c/gke-deploy/core/resource/templates/namespace.yaml)?

Unless you plan on embedding a few dozen more templates -- and even then until you actually do -- you might want to just embed the string directly, in Go code:

```go
const namespaceTemplate = `
apiVersion: v1
kind: Namespace
metadata:
  name:
`
```

If you do decide to keep packr2 as a dependency, you might want to vendor it so you can more closely control the version of packr2 that's used to embed the Go code.

cc @stewchen in case this came up during the original code review.

Reply:
Just to check whether it would work, I inlined namespace.yaml and removed the packr library dependency, which removed 258k lines of dependencies!

Reply:
In the interest of getting this in as a simple copy from the other repo, we can just merge this as-is and remove the dependency as a followup PR. Don't worry about trying to make this change in this PR.

Reply:
That's fair. I do have plans to add at least one more template, but nowhere near a dozen. I'll create a ticket for this refactoring.